MagneticField/GeomBuilder/src/buildPseudoTrap.icc

```diff
@@ -81,42 +81,41 @@ void MagGeoBuilderFromDDD::volumeHandle::buildPseudoTrap(const DDExpandedView &f
   else
     rcheck = refPlane->toGlobal(LocalPoint(x2, 0., +halfZ)).perp();
 
-  if (MagGeoBuilderFromDDD::debug) {
-    if (fabs(rcheck - fR) > 100. * epsilon) {  //FIXME!
-      cout << setprecision(10);
-      cout << "*** WARNING: Cylinder surface not centered on beam axis " << rcheck << " " << fR << " "
-           << fabs(rcheck - fR) << endl;
-      cout << setprecision(6);
+  if (debug) {
+    if (std::abs(rcheck - fR) > 100. * epsilon) {  //FIXME!
+      LogDebug("magneticfield::volumeHandle") << setprecision(10);
```

Comment:
I doubt `setprecision` works for `LogTrace`. I deleted them.

Reply:
I start being irritated about these modifications to debug messages. This PR is about migration to DD4hep, not about cleanup of debug messages, but we are now mostly talking about and changing forth and back the latter. I do need these messages and I do need them to issue messages formatted just as they were coded. If LogDebug does not support setprecision can we leave the cout as it was? I'm really unhappy about having to reformat the messages and possibly even to recode them as couts here and there in my private version (if I need formatting that LogTrace does not support) when I will need them. @slava77, I know that is the policy, but I doubt anybody else has ever run this code with debug on in the past 15 years, so I really wonder what is the purpose of messing up the PR with all these changes to debug messages.

Reply:
I did not push for switching to Log\*, however, once it was done, I was motivated to see it applied uniformly.
A large fraction of my review was in order to preserve the old formatting.

Reply:
For the files being added or revised to work with DD4hep, I replaced `cout` with `LogTrace` because there is a general recommendation to not use `cout` in code that is not in the `test` directory. The Log utilities are recommended instead. I did not change `cout` in the old code because I wanted to avoid major revisions and updating of that code.
I will try to ensure that the formatting of the debug messages is not changed by this PR.

Reply:
I was able to restore the `setprecision` statement. Note that this file is not used with the DD4hep version because we don't expect there will be any pseudo-trapezoids.